airrohr-firmware/DHT.cpp

```diff
@@ -5,17 +5,18 @@ written by Adafruit Industries
 */
 
 #include "DHT.h"
+#include "PCF8574.h"
 
 #define MIN_INTERVAL 2000
 
-PCF8574 pcf8574; //class object
+PCF8574 pcf8574(0X02); //class object
 
 DHT::DHT(uint8_t pin, uint8_t type, uint8_t address) {
   _pin = pin;
   _type = type;
   pcf8574_address = address;
 
-  pcf8574.setAddress(address);
+  //pcf8574.setAddress(address);
```

Comment:
@Warukira I am worried that we mixed up repos as the current PCF8574 has no such member class.

Reply:
@chegejames , that's why I commented it out.